homeassistant/components/adguard/__init__.py

```diff
@@ -64,6 +67,17 @@
 
     hass.data.setdefault(DOMAIN, {})[DATA_ADGUARD_CLIENT] = adguard
 
+    try:
+        version = await adguard.version()
+    except AdGuardHomeConnectionError as exception:
+        raise ConfigEntryNotReady from exception
+
+    if LooseVersion(MIN_ADGUARD_HOME_VERSION) > LooseVersion(version):
+        _LOGGER.error(
+            "This integration requires AdGuard Home v0.99.0 to work correctly"
```

Comment:
```suggestion
            "This integration requires AdGuard Home v0.99.0 or higher to work correctly"
```

Reply:
Applied on all instances of this message. Thanks! 👍